packages/docs/.vitepress/theme/Components/Layout/Features/Sidebar/Lib/UseSidebarItems.ts

```
@@ -0,0 +1,213 @@
+import type { SidebarBadge, SidebarItem } from '@/.vitepress/theme/Components/Layout/Features/Sidebar/Types/Sidebar';
+import type { ComputedRef }               from 'vue';
```

ESLint

[error] Expected "vue" (type) to come before "@/.vitepress/theme/Components/Layout/Features/Sidebar/Types/Sidebar" (internal-type).

(perfectionist/sort-imports)